cli/app.rs

```diff
@@ -913,6 +922,7 @@ In addition to standard SQL commands, the following special commands are availab
 
 Special Commands:
 -----------------
+.exit ?<CODE>              Exit this program with return-code CODE
```

Comment:
This uppercase letter `CODE`, maybe it has a special meaning. (I just copied it from SQLite)